examples/channel_service.rs

```diff
@@ -77,15 +79,16 @@ impl NewChannelService {
     }
 }
 
-impl NewService for NewChannelService {
-    type Request = String;
-    type Response = String;
-    type Error = Error;
-    type InitError = io::Error;
-    type Service = ChannelService;
-    type Future = FutureResult<Self::Service, io::Error>;
+impl Service<()> for NewChannelService {
```

Comment:
Err, this seems like an unrelated change?

Reply:
Or was it just that it wasn't tested before, and wasn't compiling now that it's tested?